src/common/AndroidUtils.ts

```diff
@@ -446,22 +418,17 @@ export class AndroidSDKUtils {
         });
     }
 
-    public static launchURLIntent(
+    public static async launchURLIntent(
         url: string,
         emulatorPort: number
     ): Promise<boolean> {
         const openUrlCommand = `${AndroidSDKUtils.getAdbShellCommand()} -s emulator-${emulatorPort} shell am start -a android.intent.action.VIEW -d ${url}`;
-        return new Promise((resolve, reject) => {
-            try {
-                CommonUtils.executeCommandSync(openUrlCommand);
-                resolve(true);
-            } catch (error) {
-                reject(error);
-            }
-        });
+        return CommonUtils.executeCommandAsync(openUrlCommand)
+            .then((result) => Promise.resolve(true))
```

Comment:
You know, this isn't necessarily explicitly related to this PR, but sniffing this scenario of "return `true` but never return `false`", I followed this all the way back up to `Preview.launchPreview()`, called from `Preview.run()` (i.e. the top of the chain), and as near as I can tell there's no reason for this Promise chain to return a `boolean`. The Promise resolution value is never read, again as near as I can tell.  Food for thought.

Reply:
yeah, I don't know why this was done this way either. I'm in favor of changing such cases to have `Promise<void>` return type. Will go ahead and make the changes.